package.json

```diff
@@ -36,8 +36,10 @@
     "storybook": "start-storybook -p 6006",
     "tag:beta:all": "ts-node ./scripts/tag-beta-all",
     "test": "jest",
+    "test:coverage": "jest --coverage",
     "tsc": "tsc",
     "validate": "yarn tsc && yarn lint && yarn test",
+    "validate:ci": "yarn tsc && yarn lint && yarn test:coverage --ci",
```

Comment:
```suggestion
    "validate:ci": "yarn validate && yarn test:coverage --ci",
```

Reply:
This would mean running the tests twice which I'm not such a fan of, but then less duplication is good so 🤷‍♂️